playbooks/cloud-provider-openstack-acceptance-test-keystone-authentication-authorization/run.yaml

```diff
@@ -153,7 +155,7 @@
 
           # Start k8s-keystone-auth webhook service
           [[ "$OS_AUTH_URL" =~ "v3" ]] && keystone_url=${OS_AUTH_URL} || keystone_url=${OS_AUTH_URL}/v3
-          nohup ./k8s-keystone-auth-amd64 \
+          nohup ./k8s-keystone-auth \
```

Comment:
I thought we will only keep binaries suffixed with `-arch` in cpo's Makefile, no?

Reply:
Currently, we allow both, which will need to change when we add multi-arch test. Since it's required to keep behavior for old and new branches, it's not that suitable to directly use -arch here as I was planed.

Reply:
OK, agree.